base/applications/mspaint/palette.h

```diff
@@ -11,6 +11,8 @@
 class CPaletteWindow : public CWindowImpl<CPaletteWindow>
 {
 public:
+    bool bk = false;
```

Comment:
This looks like something that belongs in the palette model, instead.
(model-view-controller design pattern – I tried to separate at least model and view+controller)

Reply:
Is there any class diagram in uml for this application?